btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange/expandingTheForm.cy.ts

```diff
@@ -0,0 +1,114 @@
+import payFeesForBtrRegsigin from '../../../fixtures/payFeeForBtrRegsigin.json'
+
+describe('accessibility -> Beneficial Owner Change', () => {
+  beforeEach(() => {
+    cy.intercept(
+      'GET',
+      'https://pay-api-dev.apps.silver.devops.gov.bc.ca/api/v1/fees/BTR/REGSIGIN',
+      payFeesForBtrRegsigin)
+    cy.visit('/')
+    cy.wait(1000)
+    cy.injectAxe()
+  })
+
+  it('checks page passes accessibility', () => {
+    // For now, this is an example of passing checks
+    cy.checkA11y('[data-cy=page-header]')
+    cy.checkA11y('[data-cy=page-info-text]')
+    cy.checkA11y('[data-cy=add-new-btn]')
+
+    // NB: these tests will be uncommented when their corresponding tickets are worked on
+
+    // Expanding the form - pre filled out 19442
```

Comment:
I think it makes more sense to keep this test inside the file it was in previously and inside its own 'it'. That's also what we've done for the other tests

Reply:
Updated